node/rpc-api/src/data_types.rs

```diff
@@ -24,17 +24,18 @@ use clock::ChainEpoch;
 use fil_types::{json::SectorInfoJson, sector::post::json::PoStProofJson};
 pub use forest_libp2p::{Multiaddr, Protocol};
 use forest_libp2p::{Multihash, NetworkMessage};
+use fvm_shared::bigint::{bigint_ser, BigInt};
 use ipld::json::IpldJson;
 use message::{
     message_receipt::json::MessageReceiptJson, signed_message,
     signed_message::json::SignedMessageJson, unsigned_message, SignedMessage, UnsignedMessage,
 };
 use message_pool::{MessagePool, MpoolRpcProvider};
-use num_bigint::{bigint_ser, BigInt};
+use num_bigint::bigint_ser::json;
+use num_bigint::bigint_ser::json::opt;
 use state_manager::{MiningBaseInfo, StateManager};
 use vm::{ActorState, TokenAmount};
 use wallet::KeyStore;
-
```

Comment:
You probably didn't mean to remove the newline between the imports and the code.